### DISCUSSION TITLE
Error when upgrading background-music on multiple Macs ( all with latest OS ).

### DISCUSSION THREAD

**Discussion by iTris666** [Homebrew/discussions#5552] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.14
ORIGIN: https://github.com/Homebrew/brew
HEAD: 39d51a45e89ddb4572494c845522509e3deeeb61
Last commit: 5 days ago
Core tap JSON: 12 Aug 09:01 UTC
Core cask tap JSON: 12 Aug 09:01 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 20
Homebrew Ruby: 3.3.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.4_1/bin/ruby
CPU: 20-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.46.0 => /opt/homebrew/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 14.6.1-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: 15.4
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  bokken-cli
  openssl@1.1
  python-packaging
```

### Description of issue

Here is the output I get ( I asked to type my admin password, which I do) :

`==> Upgrading background-music
==> Downloading https://raw.githubusercontent.com/Homebrew/homebrew-cask/79a400210b378c8512e0c4fce9ff27f143ce56cb/Casks/b/background-music.rb
##################################################################################################################################################################################################################################### 100.0%
==> Downloading https://github.com/kyleneideck/BackgroundMusic/releases/download/v0.4.3/BackgroundMusic-0.4.3.pkg
Already downloaded: /Users/tristan/Library/Caches/Homebrew/downloads/f26261d677adedfa65aaf9099670dfbb0a883dfa3b6a01e3ceb1a9859659e34f--BackgroundMusic-0.4.3.pkg
==> Removing launchctl service com.bearisdriving.BGM.XPCHelper
Password:
==> Uninstalling packages with sudo; the password may be necessary:
==> Removing files:
/Library/Application Support/Background Music
/Library/Audio/Plug-Ins/HAL/Background Music Device.driver
/usr/local/libexec/BGMXPCHelper.xpc
Could not kickstart service "com.apple.audio.coreaudiod": 1: Operation not permitted
==> Purging files for version 0.4.3 of Cask background-music
Error: background-music: Failure while executing; /usr/bin/sudo -E -- /bin/launchctl kickstart -kp system/com.apple.audio.coreaudiod exited with 1. Here's the output:
Could not kickstart service "com.apple.audio.coreaudiod": 1: Operation not permitted`

Doing `sudo launchctl kickstart -kp system/com.apple.audio.coreaudiod` separately fails as well. I can not uninstall/reinstall the package either because of the error.

### Relevant casks

background-music

**Comment by SMillerDev** [comment]
https://developer.apple.com/documentation/macos-release-notes/macos-14_4-release-notes#Known-Issues
